src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuotes.tsx

```diff
@@ -31,6 +49,42 @@ const motionBoxProps = {
   transition: { type: 'spring', stiffness: 100, damping: 20 },
 }
 
+const noQuotesIcon = (
```

Comment:
preferably blocking: move to separate file, see `src/components/Icons/FoxIcon.tsx` for example 🙏

Reply:
Fixed in [2c0e7de](https://github.com/shapeshift/web/pull/8087/commits/2c0e7ded59e68034b730c8481c951c1f9dbf0bfa)